spec/index.adoc

```diff
@@ -1674,6 +1674,96 @@ Any user can top-up any canister this way.
 
 This method is only available in local development instances.
 
+
+[#ic-bitcoin-canisters]
+== The IC bitcoin canisters
+
+Similar to the <<ic-management-canister,management canister>>, the Bitcoin functionality is also exposed through a facade, which we refer to as the "Bitcoin canisters".
```

Comment:
Why did we opt for making the API available trough a canister facade instead of extending the System API? Are (some of) the functions inherently asynchronous, or was it "just" making "two copies" of the API easily available for Bitcoin testnet and mainnet, respectively?

Reply:
The System API is super low level and wouldn't have nearly as nice an interface - even passing blob or text values, let alone structured values, is tedious and error prone and requires Motoko compiler support or library support in Rust.
.Receiving multiple values is even worse.

So using a canister is a good choice, IMO.